.github/workflows/devnet_deploy_contracts.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy Testnet contracts
```

Comment:
Do we really need this? Why can't we rely on the resetting the node every time? The intention for devnet is to test out infrastructure and we should be able to provision it fast and simple.